Sources/NetShears/NetShears.swift

```diff
@@ -101,10 +101,11 @@ public final class NetShears: NSObject {
                         HPACKHeadersRequest: [String: String]?,
                         HPACKHeadersResponse: [String: String]?){
         let request = NetShearsRequestModel(url: url, host: host, method: method, requestObject: requestObject, responseObject: responseObject, success: success, statusCode: statusCode, duration: duration, HPACKHeadersRequest: HPACKHeadersRequest, HPACKHeadersResponse: HPACKHeadersResponse, isFinished: true)
-        if loggerEnable {
+
+        if loggerEnable, !Ignore.shouldIgnore(netShearsRequest: request, on: .logger) {
             RequestStorage.shared.newRequestArrived(request)
         }
-        if listenerEnable {
+        if listenerEnable, !Ignore.shouldIgnore(netShearsRequest: request, on: .listener) {
```

Comment:
It would be better as
```suggestion
        if listenerEnable, Ignore.shouldIgnore(netShearsRequest: request, on: .listener) == false{
```